src/mavis/summary/summary.py

```diff
@@ -1,12 +1,14 @@
-from typing import Dict, List
+from typing import Mapping, Dict, List, Tuple
 
-from ..annotate.genomic import Transcript
+from ..annotate.genomic import BioInterval, Transcript
 from ..breakpoint import Breakpoint, BreakpointPair
 from ..constants import CALL_METHOD, COLUMNS, DISEASE_STATUS, PROTOCOL, SVTYPE
 from ..interval import Interval
-from ..pairing.pairing import pair_by_distance, product_key
+from ..pairing.pairing import equivalent, pair_by_distance, product_key
 from ..util import get_connected_components
 from .constants import PAIRING_STATE
+from ..cluster.cluster import merge_breakpoint_pairs
+from itertools import chain
```

Comment:
library imports should be above relative imports

Reply:
Addressed in [960bd67](https://github.com/bcgsc/mavis/pull/336/commits/960bd67fdca774210a6cd5aac4c8c14bbe2a2a5f)